syntax = "proto3";

service Testing
{
	rpc TestService (TestReq) returns (TestResp);
}

message TestReq
{
    string user_name = 1;
	int32 user_id = 2;
}

message TestResp
{
	string company_name = 1;
	string designation = 2;
	int32 salary_eur = 3;
}
